routers/users.py

```
@@ -0,0 +1,143 @@
+            email: Annotated[EmailStr, Form()],
+            nickname: Annotated[str, Form()],
+            password: Annotated[str, Form()],
+            password_confirm: Annotated[str, Form()]
+    ):
+        if password != password_confirm:
+            raise HTTPException(status_code=400, detail="패스워드 불일치")
+        return cls(email=email,nickname=nickname, password=password)
+
+class UserResponse(BaseModel):
+    access_token: str
+    token_type: str = "bearer"
```

Ruff (0.14.11)

Possible hardcoded password assigned to: "token_type"

(S105)